test/OpenTelemetry.Exporter.Prometheus.AspNetCore.Tests/PrometheusExporterMiddlewareTests.cs

```diff
@@ -248,6 +248,46 @@ public Task PrometheusExporterMiddlewareIntegration_UseOpenMetricsVersionHeader(
             acceptHeader: "application/openmetrics-text; version=1.0.0");
     }
 
+    [Fact]
+    public async Task PrometheusExporterMiddlewareIntegration_CanServeOpenMetricsAndPlainFormats()
+    {
+        using var host = await StartTestHostAsync(
+            app => app.UseOpenTelemetryPrometheusScrapingEndpoint());
+
+        var tags = new KeyValuePair<string, object>[]
+        {
+            new KeyValuePair<string, object>("key1", "value1"),
+            new KeyValuePair<string, object>("key2", "value2"),
+        };
+
+        using var meter = new Meter(MeterName, MeterVersion);
+
+        var beginTimestamp = DateTimeOffset.Now.ToUnixTimeMilliseconds();
+
+        var counter = meter.CreateCounter<double>("counter_double");
+        counter.Add(100.18D, tags);
+        counter.Add(0.99D, tags);
+
+        var testCases = new bool[] { true, false };
```

Comment:
add more values here. {true, false, true, true, false}. 

The issue was surfacing due to this sequence > OpenMetrics, Plaintext, OpenMetrics (Plaintext was overwriting OpenMetrics data (specifically scope information).